lib/services/pdf_renderer/index.js

```diff
@@ -24,9 +24,8 @@ const publicApi = {
   renderGameToPdf(game) {
     store.dispatch("startNewPrintJob")
 
-    const components = store.getters.findAllGameComponents(game),
+    const components = store.getters.findAllPrintableGameComponents(game),
```

Comment:
Nice work!
- an intention-revealing getter name
- part of the intention removes the null check 2 lines later

One extra thing that can now be removed is the `compact` call, which catches and removes those nulls that we're no longer returning.